newbook/02-data.Rmd

```diff
@@ -977,8 +975,9 @@ We can compare this to what we see in the NASIS Pedon Diagnostic Features table:
 
 Any profiles that have logic errors detected are stored in `soilDB.env` `bad.pedon.ids` variable after you run `fetchNASIS`:
 
+<!-- consider eval=FALSE for less output that contains very little information -->
```

Comment:
I want the output in there -- please do not remove this / remove this comment

Reply:
OK.